ostd/src/arch/x86/cpu/extension.rs

```diff
@@ -112,6 +112,7 @@ define_isa_extensions! {
         RDRAND,       Ecx(30), "The processor supports RDRAND instruction.";
 
         XAPIC,        Edx( 9), "APIC On-Chip.";
+        PAT,          Edx(16), "Supports the Page Attribute Table (PAT).";
```

Comment:
There is no need to define this, nor do I intend to. At least, we don't define "FPU" because it's available on all x86-64 CPUs.